examples/istio/main.tf

```diff
@@ -154,21 +157,19 @@ resource "helm_release" "istiod" {
   name       = "istiod"
   namespace  = helm_release.istio_base.metadata[0].namespace
   version    = local.istio_chart_version
-  wait       = false
```

Comment:
by default, our addons do not wait - instead, we rely on the normal Kubernetes contructs provided by the operators/controllers deployed by the charts to eventually reconcile their state. Lets revert this and remove the explicit depends on for other Istio resources